### PR TITLE
Feature/issues/184

### PR DESCRIPTION
**Pull request by Mizzick**

### CODE REVIEW

File: Extension/lib/content-script/assistant/js/tools.js [hunk 1]
```diff
@@ -1,131 +0,0 @@
...
-				onRadioButtonChanged();
-				$(div).css('width', el.css('width'));
-
-				$(this).addClass('ui-planeta');
-				el.hide();
-			});
-		},
-
-		checkbox: function () {
-			this.defaults = {
```
Comment (atropnikov) [inline Extension/lib/content-script/assistant/js/tools.js]: checbox is used on options.html page.

**Comment by ameshkov** [comment]
@Mizzick we still need to get rid of jquery and jquery UI.

**Comment by ameshkov** [comment]
@atropnikov this also seems to be good (not yet tested though)

**Comment by atropnikov** [comment]
@ameshkov I agree with you. We need to test this branch before merge to master.

**Comment by edgr-sanchez** [comment]
Will this branch resolve these issues?

image

**Comment by ameshkov** [comment]
@edgr-sanchez seems to be an issue specific to some website. Where have you got it?

**Comment by edgr-sanchez** [comment]
@ameshkov hmm... now that you mentioned it, I narrowed it down to it only happening when I use WordPress to edit a page while using the Divi editor (Divi is a theme from Elegant Themes). 

It only shows these errors when Adguard is enabled though. I can make them go away by going to Safari > Preferences > Extensions > AdGuard and disabling it. 

I'm assuming it's an issue with Divi though, since it doesn't happen on sites that don't have it enabled. 

I assume that implementing these changes would actually fix my issue, since I see references to jQuery 1.8.3 in the code to be removed.

We'll see, it's not a big deal right now.

**Comment by ameshkov** [comment]
I suspect that it's just some incorrect blocking, maybe AG blocks something what it should not.

You should be able to narrow it down if you examine "Logs" tab in the Safari console.
There should be some messages like "Content blocker has prevented blahblah".

I may figure what's wrong if you show me the output.

**Comment by ameshkov** [comment]
Also I am pretty sure that it should be enough to turn AG off for that wordpress website only. Just click AG icon and turn off the switch.

**Comment by edgr-sanchez** [comment]
@ameshkov I actually do have AG turned off for this specific domain, but it still gives those errors. 

My Safari console's logs tab only shows this:

`JQMIGRATE: Migrate is installed, version 1.4.0`

**Comment by ameshkov** [comment]
@edgr-sanchez huh, that's bad, means it is Adguard's content script somehow messes with the page.

Does it help to "Pause protection"? It should turn AG off globally.

And one more thing, do you know if there is a free online demo of that editor? Can't find it on their website.

**Comment by edgr-sanchez** [comment]
I tried pausing protection but it stills happens:

image1

Here it is when I disable the extension entirely:

image2

I'm on the latest beta available for Safari, I forgot to mention that on the first comment.

I don't think they have a demo of it but I can create a restricted user account on my site for you if you'd like.

**Comment by ameshkov** [comment]
@edgr-sanchez filed a bug a report on this:
https://github.com/AdguardTeam/AdguardBrowserExtension/issues/228

We'll see what can be done about it.

Forgot one more thing: which version of Safari do you use?

**Comment by edgr-sanchez** [comment]
@ameshkov awesome thank you.

Safari Version 9.1 (11601.5.17.1)

OS X 10.11.4

**Comment by ameshkov** [comment]
@Mizzick here is a list of issues with this branch.

1. When extension is in "selection" mode all underlying elements remain clickable.


2. Cursor style should be changed to "move" when it is over the drag handle:


3. "Blocking element" dialog should also be draggable

4. "LARGER" and "SMALLER" should work like "plus" or "minus" one step, not jumping to the largest and smallest


5. Slider handle must be draggable:


6. It is very hard to change the slider position now as clickable clickable region is just too small:


7. Add **box-sizing: content-box !important;** style to **.sg_border** class

8. Some other issues with dialog layout:
http://screencast.com/t/KZjYrvAun
http://screencast.com/t/czrcQjx8yL
http://screencast.com/t/L0zxOPX0Tzo4
http://screencast.com/t/745LV9Oq9O

**Comment by Mizzick** [comment]
Fixed
2. Cursor style should be changed to "move" when it is over the drag handle:
3. "Blocking element" dialog should also be draggable

**Comment by Mizzick** [comment]
Fixed
7.Add box-sizing: content-box !important; style to .sg_border class
8.Some other issues with dialog layout:

**Comment by Mizzick** [comment]
Fixed
5. Slider handle must be draggable:
6. It is very hard to change the slider position now as clickable clickable region is just too small:

**Comment by Mizzick** [comment]
Fixed
1.When extension is in "selection" mode all underlying elements remain clickable.

**Comment by ameshkov** [comment]
@Mizzick some new issues.

**1. Checkboxes are visible**

I understand that this issue is caused by oneskyapp translations. But there is a huge chance that for some languages translation won't be updated, so you should do it on your own.



**2. Wrong margins**


**3. Errors in slider widget**

**Comment by Mizzick** [comment]
so we can merge now?

**Comment by ameshkov** [comment]
@Mizzick yep, please do it

**Pull request merged**